lib/http.js

```diff
@@ -130,12 +130,7 @@ async function getManifest({manifestUrl, request}) {
   const {headers} = request;
   let response;
   try {
-    let httpsAgent;
-    if(!config['manifest-proxy'].manifestCache.secure) {
-      httpsAgent = new https.Agent({
-        rejectUnauthorized: false
-      });
-    }
+    const {httpsAgent} = brHttpsAgent;
```

Comment:
This may be a topic of future discussion but will it always be the case where we want to use the same `HttpsAgent` throughout the entire application?

Reply:
+1 for discussing elsewhere at another time.